docs/providers/spotinst/guide/serverless.yml.md

```diff
@@ -38,19 +49,26 @@ provider:
 
 functions:
   function-name:
-    runtime: nodejs4.8
+    runtime: nodejs8.3
     handler: handler.main
     memory: 128
     timeout: 30
-#    access: public
+    access: private
+#    activeVersions: [
```

Comment:
Why json format here?
Does this mean you can't define it with yaml format?

Reply:
You can, I've changed it to the correct format. Thanks!